ci-operator/config/openshift/tektoncd-pipeline-operator/openshift-tektoncd-pipeline-operator-master.yaml

```diff
@@ -61,3 +61,9 @@ tests:
   commands: make test-e2e
   openshift_installer_src:
     cluster_profile: aws
+- as: csv
+  commands: VERBOSE=2 make test-csv
+  container:
+    from: src
+  artifact_dir: /tmp/artifacts
```

Comment:
@nikhil-thomas is it working for us?

Reply:
we will know after this pr merges.
Then we can run we can validate by creating a pr to our repository